Dangerfile

```diff
@@ -67,6 +81,18 @@ has_changelog_changes = hasChangesIn(["CHANGELOG"])
 # Whether or not the LICENSE file has been modified or deleted.
 has_license_changes = didModify(["LICENSE"])
 
+# Common permutations of DO NOT SUBMIT.
+do_not_submit_variations = [
```

Comment:
Ideally anything that would cause a PR to be rejected would be checked locally when you run `scripts/check.sh`. For example we're already disallowing `@import` in [check_no_module_imports.hs](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/check_no_module_imports.sh).

Is it possible to run danger pre-push?

Reply:
Danger can be [run locally](https://danger.systems/guides/troubleshooting.html#i-want-to-work-locally-on-my-dangerfile), but not before pushing to github.

Should we keep this as a script instead so contributors don't have to push to GitHub to run the check?

Reply:
Hrm. I guess the check is checking the PR description itself, which can't be done before pushing.

Also, splitting file contents checking from the PR description seems like more trouble than it's worth.

Upon reflection this seems like it was a bad suggestion.